src/asmcnc/production/z_head_qc_jig/z_head_qc_home.py

```diff
@@ -182,6 +183,5 @@ def secret_option_c(self):
         self.sm.current = 'qcWC'
 
     def shutdown_console(self):
```

Comment:
Why have this function in the first place? just call console_utils.shutdown() directly